packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -199,6 +199,13 @@ export interface BucketImportProps {
    * @default Inferred from bucket name
    */
   bucketDomainName?: string;
+
+  /**
+   * The website URL of the bucket (if static web hosting is enabled).
+   *
+   * @default Inferred from bucket name
+   */
+  bucketWebsiteURL?: string;
```

Comment:
Please rename to `bucketWebsiteUrl`

Reply:
What's the difference between this and domainName? I also can't seem to find the actual implementation in `Bucket`.

Reply:
Thanks for the review @eladb 😄 

I've pushed the change to rename to `bucketWebsiteUrl` now.

Ah, I've just spotted that this actually will be identical to `domainName`. Good spot, I'll push a fix for that.